SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/TouchGrid.swift

```diff
@@ -13,9 +13,11 @@ import UIKit
 class TouchGrid {
     // The result of the scanning procedure will be stored in curNode and childNumber variables
     var curNode = Node()                // The node in the tree we are currently scanning
+	var previousCurNode = Node()		//	curNode during the previous scanning step
```

Comment:
Can you change `curNode` and `previousCurNode` to `scanNode` and `prevScanNode`? `curNode` worked at first, but I just realized calling it `scanNode` might make more sense since it specifies what "current" means in this context, and a variable called "previous current" is a little confusing.